src/communication/azext_communication/version.py

```diff
@@ -4,7 +4,7 @@
 # --------------------------------------------------------------------------------------------
 
 
-VERSION = '1.7.0b1'
+VERSION = '1.7.1b1'
```

Comment:
It seems a breaking change because you renamed the command group from `az communication identity` to `az communication user-identity`
```suggestion
VERSION = '2.0.0b1'
```